CHANGELOG.md

```diff
@@ -4,12 +4,20 @@
 
 ### Deprecation
 
+* **Replace optional arguments for `--password` and `--sudo-password` options with required arguments** ([#1269](https://github.com/puppetlabs/bolt/issues/1269))
```

Comment:
I think these will need to go in to a new section called `## BOLT NEXT` instead of under 1.35.0. Other than that LGTM

Reply:
Updated. Whoops.

Reply:
Whoops. Updated!